types/yeoman-generator/yeoman-generator-tests.ts

```diff
@@ -111,6 +111,7 @@ generator.option('opt4', {
 const optionsHelp: string = generator.optionsHelp();
 
 const answers: Promise<Answers> = generator.prompt([] as Questions);
+const answers2: Promise<Answers> = generator.prompt([{store: true}] as Questions);
```

Comment:
I'd add a test to ensure that plain `inquirer.Questions` can be passed to `#prompt` as well.